doc/introduction/templates.rst

```diff
@@ -152,8 +228,24 @@ randomly initialized arrays for the trainable parameters.
 .. autosummary::
     :nosignatures:
 
-    ~pennylane.init.cvqnn_layers_uniform
-    ~pennylane.init.cvqnn_layers_normal
+    ~pennylane.init.cvqnn_layers_all
+    ~pennylane.init.cvqnn_layers_theta_uniform
+    ~pennylane.init.cvqnn_layers_theta_normal
+    ~pennylane.init.cvqnn_layers_phi_uniform
+    ~pennylane.init.cvqnn_layers_phi_normal
+    ~pennylane.init.cvqnn_layers_varphi_uniform
+    ~pennylane.init.cvqnn_layers_varphi_normal
+    ~pennylane.init.cvqnn_layers_r_uniform
+    ~pennylane.init.cvqnn_layers_r_normal
+    ~pennylane.init.cvqnn_layers_phi_r_uniform
+    ~pennylane.init.cvqnn_layers_phi_r_normal
+    ~pennylane.init.cvqnn_layers_a_uniform
+    ~pennylane.init.cvqnn_layers_a_normal
+    ~pennylane.init.cvqnn_layers_phi_a_uniform
+    ~pennylane.init.cvqnn_layers_phi_a_normal
+    ~pennylane.init.cvqnn_layers_kappa_uniform
+    ~pennylane.init.cvqnn_layers_kappa_normal
```

Comment:
Instead of listing all these hear, can we just link to the `qml.init` api page, where they are all listed as well? Avoids duplication and saves with maintenance.

Reply:
I propose we do this in a separate docs PR that adapts the docs to the templates redesign.